### PR TITLE
internal: Add github example app

### PR DESCRIPTION
**Pull request by ntucker**

### CODE REVIEW

File: examples/github-app/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "github-app",
+  "version": "0.1.0",
```
Comment (ljharb) [inline examples/github-app/package.json:3]: this should probably have `"private": true`, so it's not published by accident.

**Review by ljharb — APPROVED** [review]

File: examples/github-app/package.json (hunk 1 above)
Comment (ntucker) [reply examples/github-app/package.json:3]: Line 21 has that

Comment (ljharb) [reply examples/github-app/package.json:3]: ah ok, buried in there :-)

Comment (ntucker) [reply examples/github-app/package.json:3]: Can't make it easy or it wouldn't be as fun 😆

**Pull request merged**